### DISCUSSION TITLE
[Feature] onebot-test

### DISCUSSION THREAD

**Discussion by woshiguabi** [howmanybots/onebot#80] [想法 / 建议]
希望能存在一个检测机器人实现正确支持了哪些onebot规范的测试工具

**Comment by ExerciseBook** [comment]
https://github.com/howmanybots/onebot/issues/28 传送门

**Comment by woshiguabi** [comment]
@ExerciseBook 我看过#28了，我觉得这个跟那个还不太一样，测试工具的侧重点是帮助开发者实现找出不符合规范的api，目的是方便开发者进行调试，#28的侧重点是开发者主动声明自己支持哪些api

**Comment by ExerciseBook** [comment]
啊 确实侧重不一样。了解。不知道其他协作者有什么打算。

**Comment by stdrc** [comment]
相当于需要有一个能够手动或自动地调用所有 OneBot 接口、接收所有事件、处理所有消息段的玩意，等待社区做一个😃
